packages/validator/test/cli-validator/mock-files/oas3/testoneof.yaml

```diff
@@ -1,4 +1,6 @@
 openapi: 3.0.0
+servers:
+  - url: https://oneoftester.com/api
```

Comment:
Spectral throws the craziest exception if an API def is missing a servers field lol. I plan to open an issue about it at some point